src/transform/src/map_lifting.rs

```diff
@@ -379,7 +379,7 @@ impl LiteralLifting {
                         .eval(Some(aggr.expr.eval(&[], &temp).unwrap()), &temp);
                     result.push(ScalarExpr::literal_ok(
                         eval,
-                        repr::ColumnType::new(repr::ScalarType::Bool),
+                        repr::ColumnType::new(repr::ScalarType::Bool, true),
```

Comment:
This line clashes with @benesch's other PR, and we should just fix it (here or there) to be nullable if the literal is nullable.